tests/TestUtil/Capture.cpp

```diff
@@ -118,7 +117,7 @@ static void image_stream_callback(k4a_result_t result, k4a_image_t image_handle,
     }
     else
     {
-        printf("%16d                    ", (int)image_get_timestamp_usec(image_handle));
+        printf("%16d                    ", (int)image_get_device_timestamp_usec(image_handle));
```

Comment:
Maybe we should change this to not cast to an int and print out the uint64_t?
```suggestion
        printf("%16" PRId64 "                    ", image_get_device_timestamp_usec(image_handle));
```